python/ray/autoscaler/commands.py

```diff
@@ -236,6 +236,55 @@ def attach_cluster(config_file, start, use_tmux, override_cluster_name, new):
                  override_cluster_name, None)
 
 
+def submit_cluster(config_file, screen, tmux, stop, start,
```

Comment:
Hm, `rsync_up` doesn't have `start` cluster functionality but `exec` has `start` cluster functionality. 

Ideally, you'd `start`, `rsync`, and `exec`. 

One alternative is to do `create_or_update_cluster`, `rsync`, and then `exec` but it wasn't clear to me that was the cleanest way of doing things (it would create 3 different `NodeUpdater`s)